### PR TITLE
Merge pool incentives proposals into one

### PR DESCRIPTION
**Pull request by antstalepresh**

### CODE REVIEW

File: x/pool-incentives/client/cli/tx.go [hunk 1]
```diff
@@ -17,105 +17,27 @@ import (
...
 
-func NewCmdSubmitEditPoolIncentivesProposal() *cobra.Command {
+func NewCmdSubmitUpdatePoolIncentivesProposal() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "edit-pool-incentives [potIds] [weights]",
-		Args:  cobra.ExactArgs(3),
-		Short: "Submit a edit record for pool incentives",
+		Use:   "update-pool-incentives [potIds] [weights]",
+		Args:  cobra.ExactArgs(2),
+		Short: "Submit a new record for pool incentives",
```
Comment (ValarDragon) [inline x/pool-incentives/client/cli/tx.go:40]:
```suggestion
		Short: "Submit an update to the records for pool incentives",
```

**Review by ValarDragon — APPROVED** [review]

**Review by sunnya97 — APPROVED** [review]

**Comment by sunnya97** [comment]
Tests are failing

**Comment by ValarDragon** [comment]
This is going into the pool-incentives PR, which also has failing tests, not master

**Comment by sunnya97** [comment]
Oh ok

**Review by sunnya97 — APPROVED** [review]

**Comment by sunnya97** [comment]
@antstalepresh feel free to merge whenever you're ready

**Pull request merged**